### PR TITLE
feat(aiplatform): add initial Imagen code sample and test

### PR DESCRIPTION
**Pull request by irataxy**
## Description

Fixes #355484599

- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by irataxy** [comment]
@minherz I'm not sure the tests are running. I don't see individual test cases in the log.

**Review by msampathkumar — APPROVED** [review]
LGTM

**Comment by msampathkumar** [comment]
Update: Code sample look good to me. I hoping for a second reviewer to check the repo's specific code style(if any) & merge the PR.

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String prompt = ""; // The text prompt describing what you want to see.
+
+    generateImage(projectId, location, prompt);
+  }
+
+  static PredictResponse generateImage(String projectId, String location, String prompt)
```
Comment (Sita04) [inline aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java:49]:
```suggestion
  public static PredictResponse generateImage(String projectId, String location, String prompt)
```

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String prompt = ""; // The text prompt describing what you want to see.
+
+    generateImage(projectId, location, prompt);
+  }
+
```
Comment (Sita04) [inline aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java:48]: Kindly add a method comment.

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          Path tmpPath = Files.createTempFile("imagen-", ".png");
+          Files.write(tmpPath, Base64.getDecoder().decode(bytesBase64Encoded));
+          System.out.format("Image file written to: %s\n", tmpPath.toUri());
+        }
+      }
+      return predictResponse;
+
+    } catch (ApiException e) {
+      System.out.format("Exception: %s\n", e.getLocalizedMessage());
+    }
```
Comment (Sita04) [inline aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java:95]: Don't catch the exception unless we show how to handle it. Kindly throw the exception.

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    return null;
```
Comment (Sita04) [inline aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java:96]:
```suggestion
```

**Review by Sita04 — APPROVED** [review]
Few nits. Kindly address those before merging.

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java (hunk 4 above)
Comment (irataxy) [reply aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java]: Done

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java (hunk 2 above)
Comment (irataxy) [reply aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java:48]: Done

File: aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java (hunk 3 above)
Comment (irataxy) [reply aiplatform/src/main/java/aiplatform/imagen/GenerateImageSample.java]: Done

**Pull request merged**